lib/Dist/Zilla/Plugin/InstallGuide.pm

```diff
@@ -82,6 +83,8 @@ Then install it:
 
     % make install
 
+On Windows platforms, you should use dmake or nmake, instead of make.
```

Comment:
Put backticks around the executables

Reply:
HI @doherty 

I am not sure what you mean, the endpoint is a plain text file. Issue #12 provided an example in POD and backticks are Markdown.

Please enlighten me

Reply:
Backticks applied